### PR TITLE
{lib}[foss/2019b] RDFlib v4.2.2 w/ Python 2.7.16 + 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/7094e53bfb40606fb1b21595c71b8f64 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3104.skitty.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/1ce3b6731a31f2ff3e71173076421641 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d5fd65683cac535f88f6c2fc3e96915f for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
sms - Linux centos linux 7.6.1810, AMD EPYC 7601 32-Core Processor, Python 2.7.5
See https://gist.github.com/fc6406d9aa24b9fc5698d7f23bd6fe91 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2401.golett.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/effe378b62cf5c5b9e775cbd622b9d15 for a full test report.

**Comment by migueldiascosta** [comment]
Going in, thanks @boegel!

File: easybuild/easyconfigs/r/RDFlib/RDFlib-4.2.2-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonBundle'
+
+name = 'RDFlib'
+version = '4.2.2'
+
+homepage = 'https://github.com/RDFLib/rdflib'
+description = """RDFLib is a Python library for working with RDF, a simple yet powerful language
+ for representing information."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```
Comment (boegel) [inline easybuild/easyconfigs/r/RDFlib/RDFlib-4.2.2-foss-2019b.eb]: @migueldiascosta @lexming rightfully pointed out (via Slack) that we can pull this down to `GCCcore`, so I did that in #9975...

**Pull request merged**